flagscale/serve/utils.py

```diff
@@ -23,6 +23,27 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def flatten_dict_to_args(config_dict, ignore_keys=[]):
```

Comment:
This function can be utilized by others and is better suited to move to a higher level.

Reply:
done